kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/xml/StatelessXmlLineProcessor.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ *  Copyright 2019 The Hyve
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.radarbase.connect.upload.converter.xml
+
+import org.radarbase.connect.upload.api.ContentsDTO
+import org.radarbase.connect.upload.converter.ConverterFactory
+import org.radarbase.connect.upload.converter.TimeFieldParser
+import org.radarbase.connect.upload.converter.TopicData
+import org.w3c.dom.Document
+import org.w3c.dom.Element
+
+/**
+ * Simple Processor for one line to one record of a single topic conversion.
+ */
+open class StatelessXmlLineProcessor {
```

Comment:
Perhaps rename to `XmlProcessor` and remove other `XmlProcessor` (it's not used?).